speedscale/README.md

```diff
@@ -0,0 +1,56 @@
+# Agent Check: Speedscale
```

Comment:
I'd recommend removing this line since the integration is not using the Agent

Reply:
updated